bin/export.py

```diff
@@ -321,6 +321,13 @@ def update_problems_yaml(problems):
                 }
             )
 
+    if colors:
+        for d, c in zip(data, colors):
+            color = ('' if c.startswith('#') else '#') + c
+            if d['rgb'] != color:
+                change = True
```

Comment:
Maybe check the field exists?

Reply:
The field is guaranteed to exist, thanks to lines 302 and 319 :slightly_smiling_face:

Reply:
Never mind, this is not true when the problem directory is not found, will fix!